frontend/js/encrypt.js

```diff
@@ -0,0 +1,73 @@
+function doEncrypt(el) {
+  var thresholdEl = document.getElementById('threshold');
+  var sharesEl = document.getElementById('shares');
+  
+  // validation
+  if (sharesEl.valueAsNumber <= 1) {
+    sharesEl.value = 2;
+  }
+
+  if (thresholdEl.valueAsNumber <= 1) {
+    thresholdEl.value = 2;
+  }
+
+  // ensure threshold isn't higher than shares
+  if (el && el.id === 'shares') {
```

Comment:
Here is why we have `this` passed for the callers of `doEncrypt()`

It's not really necessary though since we could just run this check every time and remove the argument...

I'll try to update this PR today so they all work without args for consistent API in the dom.